_rails/how-to-guides/deployment/systemd.md

```diff
@@ -167,6 +167,10 @@ If you'd like to specify custom process signals you can do this in [your Manifes
 
 ## Migrating from Bluepill to systemd
 
+<div class="notice notice-danger"><p>
+🚨 systemd is only supported by Ubuntu version 16.04 and up. Please do not attempt to upgrade previous versions to use systemd - it will fail.
```

Comment:
It's not that it won't work, but servers < 16.04 will continue using bluepill - this means if they try to upgrade Ruby to a version that requires bluepill later, it will fail.

Reply:
AH! I will change it. Thank you, kind sir.